monsters.py

```diff
@@ -405,7 +405,7 @@ def main():
     for m in olddata:
         mons = Monster(*m)
         old_monsters[mons.id] = mons
-    monsters = sorted(all_monsters.itervalues(), key=attrgetter('id'))
+    monsters = sorted(iter(list(all_monsters.values())), key=attrgetter('id'))
```

Comment:
This looks like simply `sorted(all_monsters.values(), key=attrgetter('id'))` should work?